doc/doxygen/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2020 by the hyper.deal authors
+##
+## This file is part of the hyper.deal library.
+##
+## The hyper.deal library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 3.0 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE.MD at
+## the top level directory of hyper.deal.
+##
+## ---------------------------------------------------------------------
```

Comment:
```suggestion
/*  ______________________________________________________________________
 *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
 *
 *  Copyright (C) 2021 by the ExaDG authors
 *
 *  This program is free software: you can redistribute it and/or modify
 *  it under the terms of the GNU General Public License as published by
 *  the Free Software Foundation, either version 3 of the License, or
 *  (at your option) any later version.
 *
 *  This program is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *  GNU General Public License for more details.
 *
 *  You should have received a copy of the GNU General Public License
 *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 *  ______________________________________________________________________
 */
```